cockatrice/src/tab_deck_editor.cpp

```diff
@@ -303,11 +311,12 @@ void TabDeckEditor::createMenus()
     dbMenu->addSeparator();
     dbMenu->addAction(aClearFilterOne);
     dbMenu->addAction(aClearFilterAll);
-#if defined(Q_OS_WIN) || defined(Q_OS_MAC)
     dbMenu->addSeparator();
+#if defined(Q_OS_WIN) || defined(Q_OS_MAC)
     dbMenu->addAction(aOpenCustomFolder);
```

Comment:
I don't know, but that's how the code was originally, I've just left it as close to the original as possible.